doc/man3/EVP_DigestInit.pod

```diff
@@ -135,11 +138,21 @@ see L<openssl_user_macros(7)>:
 
 =head1 DESCRIPTION
 
-The EVP digest routines are a high-level interface to message digests,
-and should be used instead of the digest-specific functions.
+The EVP digest routines are a high-level interface to message digests, and
+Extendable Output Functions (XOF), which should be used instead of the
```

Comment:
s/should/can/ ??

It's not mandatory or even desirable to use XOF instead of digests almost all of the time.

Reply:
It depends what 'digest specific functions' means?? Do you know?

Reply:
I still think the "should" should be weakened.

Reply:
I dont even know what that is trying to say,

Reply:
SHOULD is an RFC word.  It means do it if you can.
We SHOULD NOT be recommending XOF instead of digests ever IMO.

Reply:
reread the sentence. You are interpreting it differently to me.
What does the original text mean by 'digest specific functions' - the CTRL function?

Reply:
The point that we are interpreting it differently is a cause for language improvement IMO :)
I can see both interpretations nonetheless.

Reply:
Cant improve it till I know what the last bit means,

Reply:
IMO, "digest-specific functions" refers to functions like SHA256_Init(), SHA256_Update(), SHA256_Final().

Personally I would just end the sentence after X0F, and delete everything after it. The "digest specific functions" are already deprecated and we don't need to further refer to them here.